packages/plugin-print/src/index.js

```diff
@@ -263,13 +259,14 @@ export default () => ({
       let alignmentX;
       let alignmentY;
 
-      if (typeof text === 'object') {
+      if (typeof text === 'object' && text.text) {
```

Comment:
I think it should be : 

```
      if (typeof text === 'object' && typeof text.text === "string") {
```

since we would want : {text:''} , to not error.

Reply:
or maybe text.text != null